### DISCUSSION TITLE
Possible naming issue in

### DISCUSSION THREAD

**Discussion by pjparks** [israel-dryer/ttkbootstrap#521] [General]
Running on a Windows box hit a defeat, Line 856 of the widgets.py had img.resize((self._metersize, self._metersize), Image.CUBIC) which failed because it couldn't find the CUBIC definition. Changed it to BICUBIC and the issue went away. Not sure if it's a defect, but it allowed me to run the library/module on my windows machine. Hope this helps.

**Comment by pjparks** [comment]
Sorry didn't see the tab for Issues. I've submitted this as a defeat on the Issues tab.

**Reply by TheDemonGuard** [reply]
Issue submitted by @pjparks was #522 
> Sorry didn't see the tab for Issues. I've submitted this as a defeat on the Issues tab.
